README.md

```diff
@@ -1,27 +1,49 @@
-# README
+# This application displays joint stock companies from Poland. Here you can see data such as stock indices, shareholder list, etc.
 
-##  Website under construction
+##  Currently the site is under construction, but you can see a draft on [jsclook.herokuapp.com](https://jsclook.herokuapp.com/)
 
+App is build on Ruby 3.1.0p0
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Installation:
 
-Things you may want to cover:
+1. Download repository
 
-* Ruby version
+2. Install gem libraries: to install go to main of this repository and run from command line:
 
-* System dependencies
+       `bundle install`
 
-* Configuration
+3. App need Postgresql database, you can find it at [https://www.postgresql.org/](https://www.postgresql.org/)
 
-* Database creation
+4. When you have installed Postgresql then cerate user with password  for app, in Postgresql:
 
-* Database initialization
+     `username: jsclook`
+     `password: 'jsclookRails'`
 
-* How to run the test suite
+     user and password are set in `/config/database.yml`
 
-* Services (job queues, cache servers, search engines, etc.)
+    Information how to create user in Postgresql you can find by read Postgresql documentation.
 
-* Deployment instructions
+5.  Database creation:
 
-* ...
+    go to main of this repository and run from command line:
+
+        `bin/rails db:migrate`
+
+    app should create database and tables in postgresql.
+
+    To seed databse with sample data:
+
+        `bin/rails db:seed`
+
+    then app fill database with data from file `db/seeds.rb`
```

Comment:
I think `bin/rails db:setup` would handle all of these (you can check it). It wouldn't go thorough the migrations actually (as `bin/rails db:migrate` does), but it would simply create the tables based on `db/schema.rb` which is faster and should work the same (assuming you have properly generated `db/schema.rb` in your repo which should be true).

Reply:
Correct, `db/schema.rb` is properly generated, so I can use this way